src/courtroom.cpp

```diff
@@ -5066,34 +5072,52 @@ void Courtroom::announce_case(QString title, bool def, bool pro, bool jud,
   }
 }
 
-void Courtroom::start_clock()
+void Courtroom::start_clock(int id)
 {
-  ui_clock->start();
+  if (id < max_clocks && ui_clock[id] != nullptr)
+  {
+    ui_clock[id]->start();
+  }
```

Comment:
Forgot a check for id < 0. Same goes for the other methods.
```suggestion
  if (id < 0 || id >= max_clocks || ui_clock[id] == nullptr)
    return;

  ui_clock[id]->start();
```

Reply:
fixed